apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -251,6 +261,22 @@ public Flux<RifRecordLoadResult> processAsync(
         });
   }
 
+  /** Refreshes the current beneficiaries materialized view. */
+  private void refreshCurrentBeneficiariesView() {
+    try (EntityManager entityManager = appState.getEntityManagerFactory().createEntityManager()) {
+      try (final Timer.Context timerRefreshCurrentBeneficiaries =
```

Comment:
There's a filter to prevent these metrics from being published to CloudWatch if we don't need to create alarms off of them so we don't incur the additional costs. We can see this data in New Relic instead (unit here is nanoseconds so the value here is 0.28 seconds)
![image](https://github.com/user-attachments/assets/131aca3a-ca5b-40d6-966d-ca0af63b4a3c)